### ISSUE TITLE
Add Batch dump control to list above items

### ISSUE THREAD

**Issue by Corgano** [DarkMatterCore/nxdumptool#57]
I knew this supported batch dumping, but for the life of me I couldn't figure out how to do it. I eventually found mention of it on issue #22. Above the menu is a bunch of information, as well as a bit that says "A to select, B to cancel, [d pad symbol] to move selection, etc etc" But it does not say "X to enter batch mode" ANYWHERE in the app itself. This should be added!

**Comment by DarkMatterCore** [comment]
@Corgano It's a bug I haven't been able to reproduce myself, but I'll look into it. Thanks for opening this issue.

**Comment by DarkMatterCore** [comment]
@Corgano Please test the attached build.

[nxdumptool_20200428_t1.zip](https://github.com/DarkMatterCore/nxdumptool/files/4544704/nxdumptool_20200428_t1.zip)

**Comment by Corgano** [comment]
That fixed it! Woo!

Might I suggest a slight change to the way it's displayed? The format of [button] action [button[ action .... makes it a little hard to follow if the button before or after triggers the action.
Instead it could be [button action] [button action] [button action] Which makes it much more clear to new users, at a glance, what does what.

**Comment by DarkMatterCore** [comment]
Alright then, I'll just make a silent update since this isn't really worthy enough to bump the version number. Thanks for testing the build.

Regarding the UI change, I think it's just not worth it since I'm not touching the old codebase unless some critical bug is discovered. The entire UI is being redesigned with lvgl.

**Comment by Corgano** [comment]
Ahh fair enough. At lease now there's some indication now though. Thanks!
